ExploreASL_Initialize.m

```diff
@@ -342,14 +342,6 @@
             x.opts.bLoadData = false;
         end
         
-        % We need to check if import/defacing and processing were run separately
-        if x.opts.bImportData && ~x.opts.bProcessData
-            x.opts.bSkipBIDS2Legacy = true;
-            x.opts.bLoadData = false;
```

Comment:
On Line 817, we still need bLoadData. But that parameter has been removed here. So it crashes on Line 817.

Reply:
And it's not only there - bLoadData is used on many places before being initialized. It also crashes then in ExploreASL.m line 96.
I do not know why you have removed this from here, but you have to add it then elsewhere.

I talk about bLoadData only, bSkipBIDS2Legacy has been removed everywhere...

Reply:
Both these errors do not happen when I ran ExploreASL(path, 0, 1) but when running ExploreASL(path, 1, 0).